play-json/src/test/scala/play/api/libs/json/JsonSpec.scala

```diff
@@ -1,552 +0,0 @@
-/*
- * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
- */
-package play.api.libs.json
-
-import java.util.{ Calendar, Date, TimeZone }
-
-import com.fasterxml.jackson.databind.{ JsonNode, ObjectMapper }
-import play.api.libs.functional.syntax._
-import play.api.libs.json.Json._
-
-import scala.collection.immutable.ListMap
-
-class JsonSpec extends org.specs2.mutable.Specification {
-
-  title("JSON")
-
-  case class User(id: Long, name: String, friends: List[User])
```

Comment:
Move the tests and fixtures which are JVM specific (using either `java.time` or Jackson types) to the appropriate `jvm` code